### PR TITLE
2-add-database-functions-that-are-needed-for-basic-gui

### PR DESCRIPTION
**Pull request by soerenlemke**

### CODE REVIEW

**Comment by soerenlemke** [comment]
@vyshnav-vinod added some features. I think we need some refactoring for better read- and testability

File: chinook.db
Comment (vyshnav-vinod) [inline chinook.db:1]: Hey chinook.db is already added inside `database/testdata`

File: database/getters.go [hunk 1]
```diff
@@ -62,12 +63,57 @@ func (g *Get) TableColumns(t string) ([]TableColumn, error) {
...
 		}
-		result = append(result, TableColumn{Name: tc.Name, Type: tc.Type})
+		result = append(result, tc)
 	}
 
 	if err = rows.Err(); err != nil {
 		return nil, err
 	}
 
+	fkRows, err := g.DB.connection.Query(fmt.Sprintf("PRAGMA foreign_key_list(%s);", t))
```
Comment (vyshnav-vinod) [inline database/getters.go:73]: I guess we can move this to TableForeignKeys ().. it should modularize the code

**Comment by vyshnav-vinod** [comment]
I have made tests for the function except get table names.. After merging this, i will add a test for that
@soerenlemke

File: chinook.db
Comment (soerenlemke) [reply chinook.db:1]: youre right.. something went wrong with merging on my side

**Pull request merged**